client/dashboard/app/style.scss

```diff
@@ -35,8 +35,3 @@ a {
 #wpcom {
 	font-size: $font-size-medium;
 }
-
-h1 {
```

Comment:
Any rationale about this change?

Reply:
Modals use <h1> for title, and in the design it doesn't use the brand font. So we changed to scope it to the PageHeading h1 instead. I couldn't find any <Modal> prop to change the title to use another element.

![Screenshot 2025-05-26 at 3 18 49 PM](https://github.com/user-attachments/assets/55ad514f-7570-4dfc-88e8-a468e8e32f38)